### PR TITLE
Design overhaul

### LINKED ISSUE #74: Logout-button
There is currently no way to log out from the page. 
When a user is logged in, the "login" button should be replaced with a "logout" button. 
When clicked, the user should be logged out from the gallery.

### PR DESCRIPTION
**Pull request by rosasam**
This PR changes the look of most of the FAFA-pages. New design features include:

- New look for navbar (also mobile-friendly now. No overflow, only burger)
- Admin-specific colors
- New look for folders
- New look for titles
- New look for breadcrumb (also no overflow here on mobile)
- Logout functionality (Fixes #74 )
- Remove landing page
- Login-page design (combines the old landing page and login page)
- A lot of minor design and usability improvements

image

### CODE REVIEW

File: fotogalleri/gallery/templates/components/folder.html [hunk 1]
```diff
@@ -7,7 +7,8 @@
   data-url="/view/{{ folder.full_path }}"
   data-id="{{ folder.pk }}"
 >
-  <i class="fas fa-folder" aria-hidden="true"></i>
-  <input type="checkbox" style="display: none;" />
-  <div class="folder-text">{{ folder.path }}</div>
+{{ folder.path }}
+  <input type="checkbox" style="display: none; margin-left: 10px;" />
+  <!-- <i class="fas fa-angle-right" aria-hidden="true" style="font-size: larger;"></i> -->
```
Comment (c00k133) [inline fotogalleri/gallery/templates/components/folder.html]: Although `WIP`: should probably remove comment

File: fotogalleri/gallery/templates/components/styled_button.html [hunk 2]
```diff
@@ -4,17 +4,9 @@
...
-        <p class="text-left-aligned {{ sizes }}">
-          {{ button_text }}
-        </p>
-      </div>
-    </button>
-  </a>
-</div>
+<a href="{{ url_name|default:"javascript:;" }}">
+  {# style should be: primary | secondary | admin #}
+  <button class="button styled-button-{{style}}" type="{{ button_type }}">
```
Comment (c00k133) [inline fotogalleri/gallery/templates/components/styled_button.html]: NIT: `{{style}}` => `{{ style }}`

File: fotogalleri/gallery/templates/components/toolbar.html [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {% load static %}
 {% load view %}
 
+{% if is_admin %}
```
Comment (c00k133) [inline fotogalleri/gallery/templates/components/toolbar.html:4]: Should we indent under the clause?

**Comment by c00k133** [comment]
Looks really nice, great work!

File: fotogalleri/gallery/templates/components/styled_button.html (hunk 2 above)
Comment (rosasam) [reply fotogalleri/gallery/templates/components/styled_button.html]: fixed

File: fotogalleri/gallery/templates/components/folder.html (hunk 1 above)
Comment (rosasam) [reply fotogalleri/gallery/templates/components/folder.html]: fixed

File: fotogalleri/gallery/templates/components/toolbar.html (hunk 3 above)
Comment (rosasam) [reply fotogalleri/gallery/templates/components/toolbar.html:4]: yes, fixed

File: fotogalleri/fotogalleri/settings.py [hunk 4]
```diff
@@ -30,7 +30,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = env('DEBUG', True)
 
-ALLOWED_HOSTS = ['localhost']
+ALLOWED_HOSTS = ['localhost', '192.168.1.2', '192.168.1.255']
```
Comment (c00k133) [inline fotogalleri/fotogalleri/settings.py:33]: Why add `192.168.1.2` and `192.168.1.255`?

File: fotogalleri/gallery/templates/components/folder.html [hunk 5]
```diff
@@ -7,7 +7,6 @@
   data-url="/view/{{ folder.full_path }}"
   data-id="{{ folder.pk }}"
 >
-  <i class="fas fa-folder" aria-hidden="true"></i>
-  <input type="checkbox" style="display: none;" />
-  <div class="folder-text">{{ folder.path }}</div>
+{{ folder.path }}
```
Comment (c00k133) [inline fotogalleri/gallery/templates/components/folder.html:10]: NIT: indentation

File: fotogalleri/gallery/templates/login.html [hunk 6]
```diff
@@ -1,11 +1,44 @@
...
+            <h1 class="title hero-title ">
+              Välkommen till TF:s fotogalleri
+            </h1>
+            <h2 class="subtitle">
+              eller FAFA Alternativt FotoAlbum
+            </h2>
+          </div>
+          {% include "login_form.html" %}
+          {# TODO: uncomment this once public images are supported #}
+          <!-- <a href="/view/">
```
Comment (c00k133) [inline fotogalleri/gallery/templates/login.html:33]: Should this comment be removed?

File: fotogalleri/gallery/templates/view_images.html [hunk 7]
```diff
@@ -14,36 +16,53 @@
...
     >
-      {% if folder %}
-        <div class="card-title">
-          Mappar
-        </div>
-      {% endif %}
-
+    <div>
       {% for folder in folders %}
+      <div>
```
Comment (c00k133) [inline fotogalleri/gallery/templates/view_images.html:43]: NIT: indentation

**Review by c00k133 — APPROVED** [review]
Looks great!

+1

**Pull request merged**